cli_tools/common/utils/param/param_utils.go

```diff
@@ -47,7 +47,7 @@ func GetProjectID(mgce domain.MetadataGCEInterface, projectFlag string) (string,
 
 func populateScratchBucketGcsPath(scratchBucketGcsPath *string, zone string, mgce domain.MetadataGCEInterface,
 	scratchBucketCreator domain.ScratchBucketCreatorInterface, file string, project *string,
-	storageClient domain.StorageClientInterface) (string, error) {
+	storageClient domain.StorageClientInterface, cleanupSharedScratch bool) (string, error) {
```

Comment:
cleanupSharedScratch sounds like the whole scratch bucket is deleted. It also sounds like no other criteria is used to perform this operation, while in reality current project has to be a mismatch for scratch bucket project. 
Maybe: cleanupFileOnMismatchedProject?

Reply:
good point. Renamed to 'removeFileFromSharedScratch'

Reply:
Also consider adding docs for it so it's even clearer what it does without looking at the code.

Reply:
added docs to populateScratchBucketGcsPath